pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3828,9 +3821,7 @@ int tracepoint__sched__sched_process_exit(struct bpf_raw_tracepoint_args *ctx)
     // evaluate should_trace before removing this pid from the maps
     bool traced = should_trace(&data);
 
-    // Remove this pid from all maps
     bpf_map_delete_elem(&task_info_map, &data.context.task.host_tid);
```

Comment:
Just making sure, this stays here and shouldn't move to the `sched_process_free` program?

Reply:
Yes, this should stay here since it is tid and not pid